src/app/app-config.service.ts

```diff
@@ -21,7 +21,30 @@ class Config {
   aboutText: string = "";
 
   constructor() {
-    // Add default values to severities map?
+    this.severities.set("maintenance", {
+    start: 0,
+    end: 0,
+    color: "#50c3a5",
+    colorblind: "#e1be6a"
+    });
+    this.severities.set("operational", {
+    start: 1,
+    end: 33,
+    color: "#50c3a5",
+    colorblind: "#8ce05d"
+    });
+    this.severities.set("limited", {
+    start: 34,
+    end: 66,
+    color: "#f5c451",
+    colorblind: "#5d3a9b"
+    });
+    this.severities.set("broken", {
+    start: 67,
+    end: 100,
+    color: "#ee6a5f",
+    colorblind: "#d62c13"
+    });
```

Comment:
Indentation got lost here?

Reply:
Yeah, I'll fix it in a different PR later on. Not worth to void the review for this right now.